plugins/crd/crd_utils.go

```diff
@@ -46,31 +55,83 @@ func newCustomResourceDefinition(plugin *Plugin, FullName, Group, Version, Plura
 		GetOpenAPIDefinitions: v1.GetOpenAPIDefinitions,
 	})
 
-	crdClient := plugin.apiclientset.ApiextensionsV1beta1().CustomResourceDefinitions()
+	crdClient := plugin.apiclientset
+	// Add NSM CRD version
+	crd.ObjectMeta.Annotations = map[string]string{
+		nsmCRDVersionAnnotationKey: nsmCRDVersion,
+	}
+	// Starting with 1.11.X SpecReplicasPath and StatusReplicasPath have become mandatory fields,
```

Comment:
I don't think we've settled on requiring kubernetes 1.11.x yet, so this is good to add here.

Reply:
👍 my k8s runs the master of kubernetes so, to test PR locally I had to use.